front-end/package.json

```diff
@@ -7,10 +7,8 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "axios": "^0.19.2",
-    "bootstrap": "^4.4.1",
```

Comment:
Removed these two packages as @PaulNewling and @AndreEPaul were planning on using this but dropped it last sprint.